src/components/settings/SettingsLMSTab/tests/SAPConfig.test.jsx

```diff
@@ -166,6 +166,7 @@ describe('<SAPConfig />', () => {
     expect(enableButton).not.toBeDisabled();
   });
   test('it creates new configs on submit', async () => {
+    jest.setTimeout(30000);
```

Comment:
This applies to the whole file, we can apply the timeout to the test() function itself.
https://jestjs.io/docs/api#testname-fn-timeout